<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parkkihalliIkkuna</class>
 <widget class="QMainWindow" name="parkkihalliIkkuna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parkkihalli</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1272</width>
         <height>669</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="halliKartta">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>825</width>
            <height>600</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>825</width>
            <height>600</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="interactive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="pisteLabel">
                <property name="font">
                 <font>
                  <family>Serif</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>PISTEET:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="pisteNaytto">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>50</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="rahaLabel">
                <property name="font">
                 <font>
                  <family>Serif</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>RAHA:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="rahaNaytto">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>50</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="pelaajaLabel">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Serif&quot;;</string>
            </property>
            <property name="text">
             <string>VALITSE PELAAJA:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLCDNumber" name="lauraHP">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="digitCount">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="kyborgi2HP">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="digitCount">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="kyborgi1HP">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="digitCount">
               <number>3</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="laura">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>70</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>70</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">
background-color: transparent;
border-image: none;
background: none;
border: none;
background-repeat: none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="Kuvat.qrc">
                <normaloff>:/new/prefix1/Kuvat/laura.png</normaloff>:/new/prefix1/Kuvat/laura.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>80</width>
                <height>70</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="kyborgi2">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>70</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>70</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">
background-color: transparent;
border-image: none;
background: none;
border: none;
background-repeat: none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="Kuvat.qrc">
                <normaloff>:/new/prefix1/Kuvat/teekkariKyborgi2.png</normaloff>:/new/prefix1/Kuvat/teekkariKyborgi2.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>80</width>
                <height>70</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="kyborgi1">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>70</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>70</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">
background-color: transparent;
border-image: none;
background: none;
border: none;
background-repeat: none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="Kuvat.qrc">
                <normaloff>:/new/prefix1/Kuvat/teekkariKyborgi.png</normaloff>:/new/prefix1/Kuvat/teekkariKyborgi.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>80</width>
                <height>70</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLCDNumber" name="lauraTP">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="digitCount">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="kyborgi2TP">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="digitCount">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="kyborgi1TP">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="digitCount">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="vuoronVaihto">
            <property name="minimumSize">
             <size>
              <width>248</width>
              <height>55</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>248</width>
              <height>55</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Kuvat.qrc">
              <normaloff>:/new/prefix1/Kuvat/lopetaVuoroNappi.png</normaloff>:/new/prefix1/Kuvat/lopetaVuoroNappi.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>300</width>
              <height>55</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="aseLabel">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 12pt &quot;Serif&quot;;</string>
            </property>
            <property name="text">
             <string>KÄYTÖSSÄ OLEVA ASE:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="pistooli">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>45</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>45</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-image: url(:/new/prefix1/Kuvat/pistooli.png)</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="haulikko">
              <property name="minimumSize">
               <size>
                <width>128</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-image: url(:/new/prefix1/Kuvat/haulikko.png)</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="kivaari">
              <property name="minimumSize">
               <size>
                <width>128</width>
                <height>37</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>37</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-image: url(:/new/prefix1/Kuvat/kivaari.png)</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="suojaLabel">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 12pt &quot;Serif&quot;;
</string>
            </property>
            <property name="text">
             <string>LAURAN SUOJA:</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QLabel" name="kilpi">
            <property name="minimumSize">
             <size>
              <width>63</width>
              <height>66</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>63</width>
              <height>66</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/new/prefix1/Kuvat/kilpi.png)</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="lopetaNappi">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>55</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>55</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Kuvat.qrc">
              <normaloff>:/new/prefix1/Kuvat/palaaNappi.png</normaloff>:/new/prefix1/Kuvat/palaaNappi.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>55</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Kuvat.qrc"/>
 </resources>
 <connections/>
</ui>
